### PR TITLE
Suttaplex rewrite

### PR DESCRIPTION
**Pull request by robertmusil**

### CODE REVIEW

File: client/elements/suttaplex/sc-suttaplex-list.js [hunk 1]
```diff
@@ -1,198 +1,75 @@
...
+  };
 
-  _categoryChanged() {
-    if (!this.categoryId || !this.language) {
-      return;
-    }
-    this.$.ajax.url = `${API_ROOT}\/suttaplex\/${this.categoryId}?language=${this.language}`;
-    this.$.ajax.generateRequest();
+  get apiUrl() {
+    return `${API_ROOT}/suttaplex/${this.categoryId}?language=${this.language}`
```
Comment (Bosmanfrx) [inline client/elements/suttaplex/sc-suttaplex-list.js]:
```suggestion
    return `${API_ROOT}/suttaplex/${this.categoryId}?language=${this.language}`;
```

File: client/elements/suttaplex/sc-suttaplex-list.js [hunk 2]
```diff
@@ -1,198 +1,75 @@
...
+  };
+
+  get expansionApiUrl() {
+    return `${API_ROOT}/expansion`
```
Comment (Bosmanfrx) [inline client/elements/suttaplex/sc-suttaplex-list.js]:
```suggestion
    return `${API_ROOT}/expansion`;
```

File: client/elements/suttaplex/sc-suttaplex-list.js (hunk 1 above)
Comment (janlat) [inline client/elements/suttaplex/sc-suttaplex-list.js]:
```suggestion
    return `${API_ROOT}/suttaplex/${this.categoryId}?language=${this.language}`;
```

File: client/elements/suttaplex/sc-suttaplex-list.js (hunk 2 above)
Comment (janlat) [inline client/elements/suttaplex/sc-suttaplex-list.js]:
```suggestion
    return `${API_ROOT}/expansion`;
```

File: client/elements/suttaplex/sc-suttaplex-section-title-css.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    @apply --paper-font-headline;
+    display: flex;
+    justify-content: space-between;
+    border-top: 0;
+    margin:0;
+  }
+
+  .node-secondary-heading {
+    @apply --paper-font-title;
+    margin:0;
```
Comment (Bosmanfrx) [inline client/elements/suttaplex/sc-suttaplex-section-title-css.js]:
```suggestion
    margin: 0;
```

File: client/elements/suttaplex/sc-suttaplex-section-title-css.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    justify-content: space-between;
+    align-items: baseline;
+  }
+
+  .node-top-heading {
+    @apply --paper-font-headline;
+    display: flex;
+    justify-content: space-between;
+    border-top: 0;
+    margin:0;
```
Comment (Bosmanfrx) [inline client/elements/suttaplex/sc-suttaplex-section-title-css.js]:
```suggestion
    margin: 0;
```

File: client/elements/addons/localization-mixin.js [hunk 5]
```diff
@@ -54,48 +54,56 @@ export const Localized = base => class extends base {
   async __loadLanguage(lang) {
```
[the remark this answers is not in the document]
Comment (robertmusil) [reply client/elements/addons/localization-mixin.js]: Please read the specification of async functions

File: client/elements/addons/localization-mixin.js [hunk 6]
```diff
@@ -106,22 +114,26 @@ export const LitLocalized = base => class extends connect(store)(base) {
...
+    this._languageLoaded = false;
+    this.__resources = Object.assign(
       {},
       await this.__loadLanguage(FALLBACK_LANGUAGE),
       await this.__loadLanguage(lang)
     );
+    this._languageLoaded = true;
   }
 
   async __loadLanguage(lang) {
```
[the remark this answers is not in the document]
Comment (robertmusil) [reply client/elements/addons/localization-mixin.js]: ditto

File: client/elements/suttaplex/card/sc-suttaplex.js [hunk 7]
```diff
@@ -0,0 +1,371 @@
...
+    }
+
+    return html`
+      ${suttaplexCss}
+      
+      <paper-card class="suttaplex" id="${this.item.uid}" elevation="1">
+        <div class="compact">
+          <div class="top-row">
+            <h1 title="${this.mainHeadingTitle}">${this.mainHeading}</h1>
+
```
[the remark this answers is not in the document]
Comment (robertmusil) [reply client/elements/suttaplex/card/sc-suttaplex.js]: That's the margin-bottom of h1

**Pull request merged**